### PR TITLE
Add Chloe

### PR DESCRIPTION
**Pull request by SSlayer9**

### CODE REVIEW

File: data/mods/ssb/random-teams.ts [hunk 1]
```diff
@@ -64,11 +64,17 @@ export const ssbSets: SSBSets = {
...
+		moves: ['Nasty Plot', 'Inferno', 'Psystrike'],
+		signatureMove: 'Víðsýni',
+		evs: {spa: 252, spd: 4, spe: 252}, nature: 'Timid',
+	},
 	cleann: {
 		species: 'Dragapult', ability: 'Frisk', item: ['Life Orb', 'Dragon Fang'], gender: '',
 		moves: ['Dragon Dance', 'Dragon Darts', 'Fire Punch'],
 		signatureMove: 'Malicious Intent',
-		evs: {atk: 252, def: 4, spe: 252}, nature: 'Jolly',
+		evs: {atk: 252, def: 4, spe: 252}, nature: 'Timid',
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]: ???

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -211,7 +211,27 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Fire",
 	},
-
+	// Chloe
+	viosyni: {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
	vsni: {
```
afaik unicode is a bit tricky so leave out any non-english characters.

File: data/mods/ssb/random-teams.ts (hunk 1 above)
Comment (SSlayer9) [reply data/mods/ssb/random-teams.ts]: Oops I edited the wrong mon

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -211,7 +211,27 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+	vsni: {
+		accuracy: 55,
+		basePower: 100,
+		category: "Special",
+		desc: "No additional effect.",
+		shortDesc: "No additional effect.",
+		name: "Víðsýni",
+		pp: 5,
+		priority: 1,
+		flags: {protect:1},
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		flags: {protect: 1},
```

**Pull request merged**